core/src/main/java/com/scalar/db/api/Admin.java

```diff
@@ -415,4 +415,13 @@ void addNewColumnToTable(String namespace, String table, String columnName, Data
    */
   void importTable(String namespace, String table, Map<String, String> options)
       throws ExecutionException;
+
+  /**
+   * Returns the names of the existing namespaces. Though, only namespace containing tables are
```

Comment:
```suggestion
   * Returns the names of the existing namespaces. However, only namespaces that contain tables are
```

Reply:
Thank you. 
I revised all your comments in https://github.com/scalar-labs/scalardb/pull/2002/commits/c7d9e72c60493e4f1f6752a2410c7b7ca2426be3